book/src/development/emit_lints.md

```diff
@@ -0,0 +1,151 @@
+# Emitting a lint
```

Comment:
Ah this is about Late and Early passes, not really about lint emission.

Lint emission is for me about how to produce snippets, lint messages and suggestions.